x-pack/test/security_solution_api_integration/test_suites/entity_analytics/risk_engine/trial_license_complete_tier/asset_criticality.ts

```diff
@@ -237,7 +237,7 @@ export default ({ getService }: FtrProviderContext) => {
         );
       });
 
-      it('should only return 1 asset criticality record if per_page=1', async () => {
+      it('@skipInServerless should only return 1 asset criticality record if per_page=1', async () => {
```

Comment:
This test was not executed in the PR pipelines because it is in the `@skipInServerlessMKI` group. Now that it has been enabled, it is failing and needs to be fixed.

Reply:
Entity Analytics follow-up issue [here](https://github.com/elastic/kibana/issues/189067).